script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -63,7 +66,7 @@ I feel dizzy, all alone in the dark like this.
 -- 眩く、心を病みつかせるような、月。
 // 眩い or 眩く? I'm assuming 眩く because of the comma but this bastard loves
 // both dizzying and beautiful
-A dizzying, heart-wrenching moon.
+A moon so dazzling you could get lost in it.
```

Comment:
I don't think this is right. 心を病む is heart-wrenching or inducing mental suffering. Like the comment calls out, I'm pretty sure this is 眩く the verb, not 眩い as an adverb (note the comma)

Reply:
Hmm, I read this more along these lines
> 病みつき (やみつき) (n) (1) being addicted to; being wholly absorbed by; (2) falling ill; becoming sick; 
> 病みつく (やみつく) (v5k,vi) to be taken ill; to be addicted to; to become absorbed in; to give oneself up (to); 

And interpreted 心を病みつく would be more "so captivating that it steals your heart".
I also chose dazzling because I think you can kinda get away with it for both meanings - to be dazzled/starstruck isn't far off being dizzy. 
If those are both def wrong then I can revert, but heart-wrenching didn't really square as something I'd expect from the line.

Reply:
Admittedly, I found the definition I used on _Jisho_ because i couldn't make heads or tails of it and used google, but here's the "idiomatic" definition they have:
<img width="663" alt="image" src="https://user-images.githubusercontent.com/16092757/162592110-9aaf7c38-fe12-4c1e-9322-1f10e7a55f7a.png">

I wouldn't say it's _definitely_ wrong, but since this line is semi-important, I'd say this might be one for the one's who can actually dekiru nihongo. Might just be out of my abilities